docs/releases/1.10.txt

```diff
@@ -97,7 +97,13 @@ Minor features
 :mod:`django.contrib.contenttypes`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* :class:`~django.contrib.contenttypes.fields.GenericForeignKey` no longer
```

Comment:
Sure, I don't have a very good intuition for what qualifies as noteworthy, and what does not, so I'll just drop this section.